activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/runtime/ProcessVariablesIT.java

```diff
@@ -326,4 +336,360 @@ public void adminShouldSeeVariables() {
                                              variableCollection)).isTrue();
         });
     }
+    
+    @Test
+    public void shouldProperHandleProcessVariablesForAdmin() throws Exception {
+        //given
+        keycloakSecurityContextClientRequestInterceptor.setKeycloakTestUser("hradmin");
+        checkProcessVariables(true);
```

Comment:
Please, don't use a boolean to say if it's admin or not. Prefer to have two separated methods `checkAdminProcessVariables` and `checkProcessVariables`. It's clearer and it avoids an `if`/`else` block.

Reply:
I have tried to reduce use of if, but also I do not want to have a duplication of code... I have made some refactoring, but still I think that use of if statement is better in this test than use of another separate method..
Why use of simple if block makes a complication :-)?

Reply:
I totally agree duplication is bad! The `if` block itself is not a problem, but there are some patterns that can avoid extra `if/else` blocks, making your code more readable and easier to test. But let's keep it for now.